### ISSUE TITLE
Get mining fee in TransactionInformation

### ISSUE THREAD

**Issue by sixjac** [dgarage/NBXplorer#146]
I show sent and received transactions in my UI, is there a function to find the fee that was paid for spent outputs at least? Or dies it have to be manually calculated?

Thanks

**Comment by NicolasDorier** [comment]
You can't. You can't even manually calculate them UNLESS all the inputs come from a wallet tracked by NBXplorer.

**Comment by sixjac** [comment]
Ok thanks

**Comment by sixjac** [comment]
There must be a way to get the fee, this is standard info across wallets, I can do it by keeping a balance change record but there must be a more straight forward way. Yes the wallet is tracked by NBXplorer

**Comment by NicolasDorier** [comment]
as I said, yes you can `UNLESS all the inputs come from a wallet tracked by NBXplorer`

**Comment by sixjac** [comment]
Yes ok so how can I know the fee I paid as the sender/wallet user? Forget the receiving mining fee, thanks

**Comment by sixjac** [comment]
> For each input, you fetch the parent transaction and extract the output.

How do I get the parent transaction? This is not immediately obvious in the API.

Thanks

**Comment by NicolasDorier** [comment]
`client.GetTransaction`
